src/components/organize/tasks/TaskAssigneesList/TaskStatusSubtitle.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import mockAssignedTask from 'test-utils/mocks/mockAssignedTask';
+import { render } from 'test-utils';
+import singletonRouter from 'next/router';
+import TaskStatusSubtitle from './TaskStatusSubtitle';
+
+jest.mock('next/dist/client/router', () => require('next-router-mock'));
+
+describe('TaskStatusSubtitle', () => {
+    it('shows correct task status.', () => {
```

Comment:
Am I right in thinking that this only tests one of the statuses, and does not confirm that the correct status is displayed when ignored or completed?

Reply:
Yes, it only tests one, and only tests that it creates a notCompleted status. Should I add to test for the two others?

Reply:
@richardolsson should i add to the test you think?